packages/ifpack2/src/Ifpack2_BandedContainer_decl.hpp

```diff
@@ -398,31 +177,21 @@ class BandedContainer<MatrixType, LocalScalarType, false> :
   ///       with kl+ku superdiagonals, as we need the addtional storage for the LU decomposition.
   BandedContainer (const Teuchos::RCP<const row_matrix_type>& matrix,
                    const Teuchos::Array<Teuchos::Array<local_ordinal_type> >& partitions,
-                   const Teuchos::RCP<const import_type>& importer,
-                   int OverlapLevel,
-                   scalar_type DampingFactor);
+                   const Teuchos::RCP<const import_type>&,
+                   bool pointIndexed);
 
   BandedContainer (const Teuchos::RCP<const row_matrix_type>& matrix,
-                   const Teuchos::Array<local_ordinal_type>& localRows);
+                   const Teuchos::Array<local_ordinal_type>& localRows,
+                   bool pointIndexed);
 
   //! Destructor (declared virtual for memory safety of derived classes).
-  virtual ~BandedContainer ();
+  virtual ~BandedContainer () {}
```

Comment:
Senior devs feel free to chime in, but I would recommend avoiding inline anything.  Put it in the def. MueLu is still spending days hunting down duplicate symbols. Anything defined inside the 'decl' will get replicated inside the library for every include.  Perhaps a good rule of thumb would be to not do inlines w/out good reasoning? (This comment is based on the Xpetra/MueLu duplicate symbol work)

Reply:
@mhoemmen It would be kind of nice to just get rid of "virtual" inside the subclasses entirely. As far as I know there are no plans to make a container that inherits from TriDi/Banded/Dense/Sparse. Maybe another @trilinos/ifpack2 developer could confirm this?

Reply:
@mhoemmen @jjellio I ended up leaving it virtual but putting the def in def.hpp.